src/browser/components/MainPage.jsx

```diff
@@ -247,14 +250,20 @@ const MainPage = createReactClass({
       }
       var id = 'mattermostView' + index;
       var isActive = self.state.key === index;
+
+      var lastUrl = this.state.lastUrl;
+      if (this.state.lastUrl === null || this.state.lastUrl === '' || !this.state.lastUrl.includes(team.url)) {
```

Comment:
`lastUrl` is not stored for each teams. So the feature doesn't work for hidden tabs.